### PR TITLE
Adds new section pre-conference, tries to fix poster alpha sort

### PR DESCRIPTION
**Pull request by emb03**
@z3cka I will merge to stage

### CODE REVIEW

File: themes/bootstrapndlc/templates/posters.html.twig [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% extends 'partials/base.html.twig' %}
-{% set posters = page.collection() %}
+{% set sessionsF = page.collection() %}
```
Comment (z3cka) [inline themes/bootstrapndlc/templates/posters.html.twig:2]: @emb03 is `sessionsF` a typo?

Comment (emb03) [inline themes/bootstrapndlc/templates/posters.html.twig:2]: @z3cka Yay, I fixed it, this is just going to stage for now. Thanks for checking on me!

Comment (emb03) [inline themes/bootstrapndlc/templates/posters.html.twig:2]: @z3cka Finally, posters are sorting by alpha.

Comment (z3cka) [inline themes/bootstrapndlc/templates/posters.html.twig:2]: Woot!

**Pull request merged**